Makefile

```diff
@@ -6,7 +6,7 @@ PROJECT_REPO := github.com/crossplane/$(PROJECT_NAME)
 
 PLATFORMS ?= linux_amd64 linux_arm64
 
-CODE_GENERATOR_COMMIT ?= v0.2.1
+CODE_GENERATOR_COMMIT ?= c4592885b843b2f0a316718a15349ea07b3a50b1
```

Comment:
How do we feel about pinning to a commit rather than a specific release? It looks we just missed out on a code-generator release yesterday.

Reply:
I think that's fine.

Reply:
`v0.2.2` has been released and includes the recent crossplane changes :) https://github.com/aws-controllers-k8s/code-generator/releases/tag/v0.2.2